### ISSUE TITLE
Javassist MD5 mismatch

### ISSUE THREAD

**Issue by trixpan** [OpenTSDB/opentsdb#928]
When trying to build 2.3.0 I see:

```
error: invalid checksum for third_party/javassist/javassist-3.18.1-GA.jar
error: expected: 5bb83868c87334320562af7eded65cc2
error:    found: a170b370a10d2f36e5adceb057f82067
```

I checked central and indeed the MD5s mismatch. Anyone else seen this?

Cheers

**Comment by trixpan** [comment]
@tsuna - I lso checked the MD5 hash from different sources and it seems like it has changed

**Comment by paulsr87** [comment]
Facing the same issue.
Anyone going to merge the mentioned fix soon?
Thanks.

**Comment by ruchitalodha** [comment]
SAme issue

**Comment by gsempe** [comment]
We have the same blocking issue.

**Comment by manolama** [comment]
That's ugly. I think we should bump to 3.21.0-GA so that we avoid hashing issues with mirrors. I'll wait a day for https://github.com/OpenTSDB/opentsdb/pull/927 to be updated and if not make the tweak tomorrow and push to the branches.

**Comment by manolama** [comment]
Merged in b81ef90b6777295e780bbd38ca541f26e97b3648 thanks to @goll. Also merged in the other branches. Will be part of the next releases.
